extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -696,6 +705,43 @@ export class SubscriptionService {
     return null;
   }
 
+  maybeStartDeferredAccountFlow_() {
+    console.log("hjere");
+    const localService = this.platformConfig_['services'].filter(
+      (service) => 'local' === (service.serviceId || 'local')
+    );
+
+    const matchedServiceConfig = userAssert(
+      localService[0],
+      'No matching services for local found'
+    );
+    if (
+      !matchedServiceConfig['hasAssociatedAccountUrl'] ||
+      !matchedServiceConfig['accountCreationRedirectUrl']
+    ) {
+      return;
+    }
+    console.log("hjere2");
+    this.platformStore_.getGrantEntitlement().then((entitlement) => {
+      if (entitlement.source == 'local') {
+        return;
+      }
+      console.log(entitlement.source);
+      const remotePlatform = this.platformStore_.getPlatform(
+        entitlement.source
+      );
+      console.log(remotePlatform);
+      
+      const deferredAccountFlow = new DeferredAccountFlow(
```

Comment:
Should skip if we have an entitlement from the local auth endpoint that unlocks that is in the case where both google and publisher endpoint return auth since by definition the user has an unlocking publisher account.